templates/prompt-for-mfa-u2f.php

```diff
@@ -4,20 +4,66 @@
 $this->includeAtTemplateBase('includes/header.php');
 
 ?>
-<form action="<?= htmlentities($this->data['formTarget']); ?>">
+<script src="<?=SimpleSAML_Module::getModuleURL('mfa/u2f-api.js');?>"></script>
+<script type="application/javascript">
+    window.onload = function() {
+
+      console.log('data:');
```

Comment:
Did you mean to leave these console.log calls in here?

Reply:
eh, its the dev theme stuff so its useful for debugging, but material theme probably wont dump as much out